packages/rest/src/writer.ts

```diff
@@ -59,17 +60,16 @@ export function writeResultToResponse(
  */
 export function writeErrorToResponse(response: Response, error: Error) {
   const e = <HttpError>error;
-  const statusCode = (response.statusCode = e.statusCode || e.status || 500);
   if (e.headers) {
     // Set response headers for the error
     for (const h in e.headers) {
       response.setHeader(h, e.headers[h]);
     }
   }
   // Build an error object
-  const errObj: Partial<HttpError> = {
-    statusCode,
-  };
+  const errObj = buildResponseData(e, {});
```

Comment:
Should we still keep the type? `Partial<HttpError>`

Reply:
I think yes. Let me verify if the given data does in fact fit the `HttpError` type